script/add_current_epoch.rb

```diff
@@ -45,6 +45,8 @@ def solana_rpc_client(network)
     next
   end
   
+  next unless block_time
```

Comment:
this should be in a different PR

Reply:
Ok, placing it in https://www.pivotaltracker.com/story/show/183862308